cirq-google/cirq_google/api/v2/device.proto

```diff
@@ -30,12 +33,55 @@ message DeviceSpecification {
   string developer_recommendations = 4;
 }
 
+// Replaces `GateDefinition`.
+message GateSpecification {
+  // This defines the approximate duration to run the gate on the device,
+  // specified as an integer number of picoseconds.
+  int64 gate_duration_picos = 1;
+
+  // Valid targets that this gate can use.
+  // Values in this list correspond to the name of the TargetSet
+  // If unset, all combinations of `DeviceSpecification.valid_qubits` are
+  // allowed.
+  repeated string valid_targets = 2;
+
+  // This specifies the gate type and gate parameter constraints for this
+  // GateSpecification.
+  oneof gate {
+    Sycamore syc = 16;
```

Comment:
Will this micro optimization really save that much space?  Maybe one byte per gate supported, and we will support at most ~10 gates?  I am not sure the confusion is worth it.

In any case, a comment should be somewhere explained that the field numbers are reserved.

Reply:
Sounds good, will make it sequential